### PR TITLE
Refactor node and service atoms

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

File: tendrl/commons/objects/node/atoms/check_node_up/__init__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import os
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import BaseAtom
+from tendrl.commons.objects.node import Node
+
+
+class CheckNodeUp(BaseAtom):
+
+    obj = Node
```
Comment (r0h4n) [inline tendrl/commons/objects/node/atoms/check_node_up/__init__.py]: you dont need to assign obj, It is done by the commons framework automatically on registering this NS

File: tendrl/commons/objects/node/atoms/cmd/__init__.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.commons.objects import BaseAtom
+from tendrl.commons.utils.cmd_utils import Command
+from tendrl.commons.objects.node import Node
+
+
+class Cmd(BaseAtom):
+    obj = Node
```
Comment (r0h4n) [inline tendrl/commons/objects/node/atoms/cmd/__init__.py]: obj not required

File: tendrl/commons/objects/service/atoms/check_service_status/__init__.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import os
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import BaseAtom
+from tendrl.commons.objects.service import Service
+
+
+class CheckServiceStatus(BaseAtom):
+    obj = Service
```
Comment (r0h4n) [inline tendrl/commons/objects/service/atoms/check_service_status/__init__.py]: not required

File: tendrl/commons/objects/definition/master.yaml [hunk 4]
```diff
@@ -88,6 +88,20 @@ namespace.tendrl:
...
+          enabled: true
+          inputs:
+            mandatory:
+              - Node.fqdn
+              - Service.name
+          outputs:
+            - status
+          name: "check whether the service is running"
+          help: "check whether the service is running"
+          run: tendrl.commons.objects.Service.atoms.CheckServiceStatus
```
Comment (anmolbabu) [inline tendrl/commons/objects/definition/master.yaml]: Done

File: tendrl/commons/objects/node/atoms/check_node_up/__init__.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/commons/objects/node/atoms/check_node_up/__init__.py]: Done

File: tendrl/commons/objects/node/atoms/cmd/__init__.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/commons/objects/node/atoms/cmd/__init__.py]: Done

File: tendrl/commons/objects/service/atoms/check_service_status/__init__.py (hunk 3 above)
Comment (anmolbabu) [inline tendrl/commons/objects/service/atoms/check_service_status/__init__.py]: Done

File: tendrl/commons/objects/node/atoms/check_node_up/__init__.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Checking if node %s is up" %
+                    self.parameters.get("fqdn"),
+                },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id'],
+            )
+        )
+        response = os.system("ping -c 1 " + fqdn)
```
Comment (r0h4n) [inline tendrl/commons/objects/node/atoms/check_node_up/__init__.py]: use commons cmd utils for this

File: tendrl/commons/objects/service/atoms/check_service_status/__init__.py [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                    "message": "Checking status of service %s on node %s" % (
+                        service_name,
+                        self.parameters.get("fqdn")
+                    )
+                },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id'],
+            )
+        )
+        response = os.system("systemctl status %s" % service_name)
```
Comment (r0h4n) [inline tendrl/commons/objects/service/atoms/check_service_status/__init__.py]: Please use https://github.com/Tendrl/commons/blob/develop/tendrl/commons/utils/service_status.py

File: tendrl/commons/objects/node/atoms/check_node_up/__init__.py (hunk 5 above)
Comment (anmolbabu) [inline tendrl/commons/objects/node/atoms/check_node_up/__init__.py]: Done

File: tendrl/commons/objects/service/atoms/check_service_status/__init__.py (hunk 6 above)
Comment (anmolbabu) [inline tendrl/commons/objects/service/atoms/check_service_status/__init__.py]: Done

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat Please review

**Comment by anmolbabu** [comment]
@r0h4n @shtripat @nthomas-redhat @nnDarshan Please review

**Pull request merged**